src/halmos/__main__.py

```
@@ -299,14 +299,17 @@ def setup(
             if args.debug: print('\n'.join(setup_bounded_loops))
 
         setup_exs = []
+
         for idx, setup_ex in enumerate(setup_exs_all):
             if setup_ex.current_opcode() in [EVM.STOP, EVM.RETURN]:
                 setup_ex.solver.set(timeout=args.solver_timeout_assertion)
                 res = setup_ex.solver.check()
                 if res != unsat:
                     setup_exs.append(setup_ex)
+            elif args.debug:
+                print(color_warn(f'setup execution finished with {mnemonic(setup_ex.current_opcode())} (error={setup_ex.error})'))
```

setup_ex errors were hidden, even with --debug